packages/terra-toggler/docs/README.md

```diff
@@ -0,0 +1,42 @@
+# Terra Toggler
+
+Toggler component that transitions content in and out.
```

Comment:
This component can be used outside of the collapse component. @tbiethman mentioned wanting to use something like this for collapsible tree views.